libethereum/Executive.cpp

```diff
@@ -157,8 +157,8 @@ Executive::Executive(Block& _s, LastHashes const& _lh, unsigned _level):
 {
 }
 
-Executive::Executive(State& _s, Block const& _block, unsigned _txIndex, BlockChain const& _bc, unsigned _level):
-	m_s(_s = _block.fromPending(_txIndex)),
+Executive::Executive(State& o_s, Block const& _block, unsigned _txIndex, BlockChain const& _bc, unsigned _level):
```

Comment:
In some sense it's not really used as input, it is assigned immediately another State in `createIntermediateState`. On the other hand it initializes `m_state` member... Ok, I think I'll rename it